corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -586,6 +586,74 @@ def test_case_tile_pull_down(self):
             "./entry/session"
         )
 
+    def test_inline_case_detail_from_another_module(self):
+        def ensure_module_session_datum_xml(detail_inline_attr, detail_persistent_attr):
```

Comment:
Extracting a common method.
They just had different arguments since other didnt need `detail_inline_attr` so just kept it under the particular test.